File: mario.py
try:
    n = int(input("Enter an integer between 1 and 8:"))
    if n > 0 and n < 9:
        pass
    else:
        raise ValueError("Not an integer between 1 and 8")
except ValueError:
    print("Not an integer between 1 and 8")
    n = int(input("Enter an integer between 1 and 8:"))

for i in range(n):
    for printspace in range(n-i-1):
        print(" ", end="")
    for printhash in range(i+1):
        print("#", end="")
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
int n;
do
{
n = get_int("Enter an integer between 1-8: ");
}
while (n < 1 || n > 8);

for(int i = 0;i < n;i++)
{
    for(int j=0;j < n + i + 3;j++)
    {
        if (j == n || j == n + 1 || j + i < n - 1)
        {
            printf(" ");
        }
        else
        {
            printf("#");
        }
    }
    printf("\n");
}
}
